students/rob_sanchez/Lesson 3/mailroom.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import sys
+
+# List of donors and donation amounts
+donor_dict = {"Tom Cruise": {
+                "donations": [
+                  {"donation-number": '1', "amount": '100'},
+                  {"donation-number": '2', "amount": '200'},
+                  {"donation-number": '3', "amount": '300'}
+                ]},
+              "Michael Jordan": {
+                "donations": [
+                    {"donation-number": '1', "amount": '1300'}
+                ]},
+              "Katy Perry": {
+                "donations": [
+                    {"donation-number": '1', "amount": '4500'},
+                    {"donation-number": '2', "amount": '1500'}
+                ]},
+              "Adam Sandler": {
+                "donations": [
+                    {"donation-number": '1', "amount": '500'},
+                    {"donation-number": '2', "amount": '2400'}
+                ]}
+              }
```

Comment:
lesson 3 mailroom was technically supposed to use a list for data structure so I guess you jumped ahead 😉 

I would recommend to consider alternate data structure:
`{"Katy Perry": [500, 2400]}`
this should really simplify your code; since you're only storing donations, no need to create a dictionary with _only_ ever having one key. also, `donation-number` is kind of redundant as you will be able to easily get size via `len(donor_dict[name])`